hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/RecordListBuilder.java

```diff
@@ -55,6 +56,33 @@ public RecordListBuilder(@NonNull final SingleTransactionRecordBuilderImpl recor
         recordBuilders.add(recordBuilder);
     }
 
+    /**
+     * Returns the main record builder
+     *
+     * @return the main record builder
+     */
+    public SingleTransactionRecordBuilderImpl mainRecordBuilder() {
+        return childRecordBuilders().get(0);
+    }
+
+    /**
+     * Returns an unmodifiable {@link List} of all preceding record builders.
+     *
+     * @return all preceding record builders
+     */
+    public List<SingleTransactionRecordBuilderImpl> precedingRecordBuilders() {
+        return Collections.unmodifiableList(precedingRecordBuilders);
+    }
+
+    /**
+     * Returns an unmodifiable {@link List} of all child record builders.
+     *
+     * @return all child record builders
+     */
+    public List<SingleTransactionRecordBuilderImpl> childRecordBuilders() {
```

Comment:
The method's implementation is the same as `precedingRecordBuilders`. Is this on purpose?

Reply:
No, good catch! It was a copy-and-paste error. Fixed.